solidity/test/rewards/TestRewards.js

```diff
@@ -484,8 +484,22 @@ describe('Rewards', () => {
         })
     })
 
+    describe("receiveRewards", async () => {
+        it("lets closed keeps claim the rewards from multiple keeps correctly", async () => {
+            let timestamps = testValues.rewardTimestamps
+            await createKeeps(timestamps)
+            await rewards.setCloseTime(timestamps[2])
+            let rewardsReceivingKeeps = [0, 1]
+            await rewards.receiveRewards(rewardsReceivingKeeps, { from: aliceBeneficiary })
```

Comment:
Can we use the pattern with `.methods` here for consistency?

Reply:
Done.